src/anyon/anyon.jl

```diff
@@ -282,6 +282,14 @@ function get_metadata(client::Client, qpu::UnionAnyonQPU)::Metadata
         output["excluded_positions"] = Vector{Int}()
     end
 
+    if haskey(machineMetadata, "excludedCouplers")
```

Comment:
To be consistent in the Thunderhead API with `disconnectedQubits`, lets make this `disconnectedCouplers`